.prettierignore

```diff
@@ -1 +1,2 @@
 CHANGELOG.md
+.github/workflows/publish.yml
```

Comment:
prettier was mangling this line: `github.event.pull_request.title == 'Update version and CHANGELOG for release'`